agent/functional_tests/testdata/taskdefinitions/agent-introspection-validator/task-definition.json

```diff
@@ -0,0 +1,24 @@
+{
+  "family": "ecsftest-agent-introspection-validator",
+  "networkMode": "host",
+  "containerDefinitions": [{
+    "image": "amazon/amazon-ecs-agent-introspection-validator:make",
+    "name": "agent-introspection-validator",
+    "memory": 50,
+    "healthCheck": {
+        "command": ["CMD-SHELL", "echo hello"],
+        "interval": 5,
+        "timeout": 5,
+        "retries": 2,
+        "startPeriod": 1
+    },
+    "logConfiguration": {
+        "logDriver": "awslogs",
```

Comment:
why are we configuring this task to use awslogs?

Reply:
Because the logs in `misc/agent-introspection-validator/agent-introspection-validator.go` needs be to be uploaded to Cloud Watch.